### PR TITLE
boards/stm32l0: introduce common clock configuration and apply it to related boards

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR introduce a new clock configuration for stm32l0. This configuration is now used by all stm32l0 based boards: b-l072z-lrwan1, nucleo-l0xxx, lsn50, i-nucleo-lrwan1.

### Testing procedure

A green Murdock should be enough but a motivated reviewer can also test on modified boards.

### Issues/PRs references

Based on #11323

### CODE REVIEW

File: boards/common/stm32/include/l0/cfg_clock_32_16_1.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_stm32
+ * @{
+ *
+ * @file
+ * @brief       Configure STM32L0 clock to 32MHz with LSE
```
Comment (fjmolinas) [inline boards/common/stm32/include/l0/cfg_clock_32_16_1.h]: The wording here makes me think 32Mhz are achieved by using the LSE which is incorrect. Maybe reword to
`Configure STM32L0 clock to 32MHz using PLL with LSE`, what do you think?

Comment (aabadie) [reply boards/common/stm32/include/l0/cfg_clock_32_16_1.h]: Updated and directly squashed

**Comment by fjmolinas** [comment]
GO!
